mount/mode/mode.go

```diff
@@ -0,0 +1,80 @@
+package mode
```

Comment:
Not sure what you mean?  If you mean to not have mode be a subdir of mount, that was my original design, but I ended up with a duplicate import in label.  It can't import mount, since mount imports label.